contrib/android/buildozer_qml.spec

```diff
@@ -138,11 +139,12 @@ android.add_jars = .buildozer/android/platform/*/build/libs_collections/Electrum
 
 # (list) List of Java files to add to the android project (can be java or a
 # directory containing the files)
-android.add_src = electrum/gui/kivy/data/java-classes/
+#android.add_src = electrum/gui/kivy/data/java-classes/
 
-android.gradle_dependencies = me.dm7.barcodescanner:zxing:1.9.8
+#android.gradle_dependencies = me.dm7.barcodescanner:zxing:1.9.8
+android.gradle_dependencies = com.android.support:support-compat:28.0.0
 
-android.add_activities = org.electrum.qr.SimpleScannerActivity
+#android.add_activities = org.electrum.qr.SimpleScannerActivity
```

Comment:
This now commented out zxing stuff was not used prior to this PR either, right?

Reply:
Ah I see you clarified this in the relevant commit.
I've pushed just this change separately to master: https://github.com/spesmilo/electrum/commit/c3f16615ca167eb6e63415f84a32592285af5da3

Reply:
I restored the behavior from https://github.com/accumulator/python-for-android/commit/3a52f805bec49a6818d527286ce5fd09823f738a, so it is as in current releases.